application.py

```diff
@@ -4,7 +4,7 @@
 from flask import Flask
 from werkzeug.serving import WSGIRequestHandler
 
-from app import create_app
+from app import create_app, socketio  # noqa: F401
```

Comment:
Is it just a matter of importing it in itself, or might there need to be some instantiation or loading of a config as well?